lib/CheckToggle/styles.scss

```diff
@@ -57,8 +70,8 @@ $borderColour: #F8FBFC;
 /*  New switch */
 .toggle-switch + .toggle-switch__label {
   padding: 2px;
-  width: 53px;
-  height: 33px;
+  width: 34px;
```

Comment:
are we ok to change the width and height for these everywhere?

Reply:
👍 Checked everywhere that the CheckToggle is used and non were using the default size (the default was MASSIVE 😱)

Reply:
👍 Checked this and we aren't using the default size anywhere. It was MASSIVE 😱